train.py

```diff
@@ -0,0 +1,117 @@
+import os
+import time
+
+import hydra
+import torch
+import psutil
+import imageio
+import numpy as np
+from omegaconf import DictConfig
+from torchvision.utils import save_image
+
+import wandb
+from utils import get_interpolations
+from models.autoencoder import Autoencoder
+from train_utils import test_epoch, train_epoch
+
+
+def initialize_model(args: DictConfig):
+    if args.model.type == "AE":
+        model = Autoencoder(args)
+    else:
+        raise NotImplementedError("Model type not implemented")
+    return model
+
+
+def train(args: DictConfig, model: torch.nn.Module):
+    os.makedirs(args.logger.results_path, exist_ok=True)
+    os.makedirs(args.logger.checkpoint_path, exist_ok=True)
+
+    torch.manual_seed(args.system.seed)
+
+    train_autoencoder(args, model)
+    draw_interpolation_grid(args, model)
+
+
+def train_autoencoder(args: DictConfig, autoenc):
```

Comment:
```suggestion
def train_autoencoder(args: DictConfig, autoencoder: Autoencoder):
```

Use typehints as much as possible and don't be cheap on letters :)